src/lib/notificationHandlerWebsocket.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
```

Comment:
Long term -- or even soon -- we may want to have a `notificationHandlerAMQP.js` for robust notifications. The config would tell which notification backend is to be used, hence the name.